### ISSUE TITLE
Feature request: Introduce AnyReleaseBefore and multiple --package [PACKAGE NAME]

### ISSUE THREAD

**Issue by teroneko** [taori/NugetUnlister#9]
Any idea to support unlisting PrereleaseBefore and ReleaseBefore in one command?

Proposal:
```
nuget-unlist drop AnyReleaseBefore [YOUR PACKAGE NAME] [0.1.3] [..]
```

Currently I have to do something like that:

```
for AnyReleaseBefore in PrereleaseBefore ReleaseBefore; do for PackageName in PackageX PackageY; do nuget-unlist drop $AnyReleaseBefore $PackageName 0.1.3; done; done
```

Not very nice. Haha. Would be nice to support something like `nuget-unlist drop AnyReleaseBefore [..]` and multiple `--package [Package NAME]`.

**Comment by taori** [comment]
Hi, Thanks for your feature request

Unfortunately i cannot really see an immediate need to work on this. 

Usually this tool is used from within automation scripts which are written once and copy pasted past that.

The multiple packages part i would not implement at all because it acts contrary to nugets cli, which afaik does not allow for management of multiple packages either.

Regarding AnyReleaseBefore however i am willing to accept a pull request if you want to invest the time for it. It should be fairly simple to implement really.

Unless you want to do a PR i will probably take a while to get arround to working on this though.

Best regards

**Comment by teroneko** [comment]
I have created a pull request. See #10.

**Comment by taori** [comment]
Thank you. Once the pipeline issues are resolved i will merge it

**Comment by teroneko** [comment]
Another pull request has been applied. When this does not succeed for whatever reason I can only advice you to add my fork as downstream make the changes you may talk about and apply and accept pull request by yourself.

**Comment by taori** [comment]
Fixed by #11

**Comment by taori** [comment]
@teroneko Ok, the tool can be upgraded now through dotnet cli. Thanks again.
